data/json/recipes/other/tool.json

```diff
@@ -177,5 +177,26 @@
     "book_learn": [ [ "mag_survival", 2 ], [ "textbook_survival", 1 ], [ "cookbook", 2 ] ],
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "can_food_unsealed", 2 ], [ "scrap", 4 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "razor_blade",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "skills_required": [ "fabrication", 3 ],
```

Comment:
No need to specify same skill twice in skill_used in skills_required.

Reply:
`skill_used` decides what skill is trained and used for success checks, `skills_required` defines what skill level the recipe is unlocked at.